src/opnsense/service/templates/OPNsense/Dnsmasq/dnsmasq.conf

```diff
@@ -0,0 +1,133 @@
+# DO NOT EDIT THIS FILE -- OPNsense auto-generated file
```

Comment:
all the more reason for rc.d since the backend doesn't see this file and it may be missing on startup / using the old one?

Reply:
we should be safe here, https://github.com/opnsense/core/blob/master/src/etc/rc.syshook.d/early/15-templates generates the file before anything wakes up.

Reply:
there was a bug with configd regarding early start. it could still be happening. worst case I'll find out when testing images ;(

Reply:
I haven't seen the bug in ages to be honest, not sure if it makes sense to force another template reload just in case, but it's not the end of the world either if it makes you feel better.